.github/workflows/test.yml

```diff
@@ -8,8 +8,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: ['23.3']
-        elixir: ['1.12.2']
+        otp: ['23.3', '24.2']
+        elixir: ['1.12.2', '1.13.1']
```

Comment:
I'm only going to run on the latest, i.e. removing 1.12. I'm not sure I need both otp version either